src/components/Contacts/ContactDetails/ContactDetailsTab/ContactDetailsTab.graphql

```diff
@@ -1,6 +1,7 @@
 query ContactDetailsTab($accountListId: ID!, $contactId: ID!) {
   contact(accountListId: $accountListId, id: $contactId) {
     id
+    locale
```

Comment:
Are `locale` and `churchName` still needed in this query? If they are only needed for the other section, I think they should only be in the fragment. But if something else needs them, keep them.